packages/beacon-node/src/network/reqresp/beaconBlocksMaybeBlobsByRange.ts

```diff
@@ -29,6 +30,7 @@ export async function beaconBlocksMaybeBlobsByRange(
   request: phase0.BeaconBlocksByRangeRequest,
   currentEpoch: Epoch,
   partialDownload: PartialDownload,
+  metrics?: Metrics | null,
```

Comment:
```suggestion
  metrics: Metrics | null,
```
this ensures we don't accidentally omit passing metrics